models/vrp.rb

```diff
@@ -79,12 +79,13 @@ def self.create(hash, options = {})
     def as_json(options = {})
       hash = super(options)
 
-      service_ids = hash[:services]&.map{ |service| service[:id] }
+      service_ids = hash[:services]&.map{ |service| service[:id] } || []
       # Remove unknown linked_service_ids from relations
       hash[:relations]&.delete_if{ |relation|
         relation[:linked_service_ids]&.delete_if{ |id| service_ids.exclude?(id) }
```

Comment:
While we are at it, shall we do the same correction for vehicles as well?
```suggestion
      service_ids = hash[:services]&.map{ |service| service[:id] } || []
      vehicle_ids = hash[:vehicles]&.map{ |vehicle| vehicle[:id] } || []
      # Remove unknown linked_service_ids from relations
      hash[:relations]&.delete_if{ |relation|
        relation[:linked_service_ids]&.delete_if{ |id| service_ids.exclude?(id) }
        relation[:linked_vehicle_ids]&.delete_if{ |id| vehicle_ids.exclude?(id) }
```